test/DGmethods/Horizontal_integral_test.jl

```diff
@@ -0,0 +1,98 @@
+using MPI
```

Comment:
Indenting the file would make it easier to review.

Reply:
Absolutely, will do.